backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -556,10 +551,27 @@ public FutureExperimentRunDAO(
                 ModelDBActionEnum.ModelDBServiceActions.READ,
                 ModelDBResourceEnum.ModelDBServiceResourceTypes.PROJECT)
             .thenApply(
-                projIds ->
-                    new QueryFilterContext()
-                        .addCondition("experiment_run.project_id in (<authz_project_ids>)")
-                        .addBind(q -> q.bindList("authz_project_ids", projIds)),
+                resources -> {
+                  List<Resources> accessAllResources =
+                      resources.stream()
+                          .filter(Resources::getAllResourceIds)
+                          .collect(Collectors.toList());
+                  if (!accessAllResources.isEmpty()) {
```

Comment:
Please use the new field https://github.com/VertaAI/protos-all/blob/master/protos/protos/private/uac/RoleService.proto#L136 to be extremely clear that we're in this condition. Way better than making assumptions that we'll remember, because all evidence points towards us not remembering.

Reply:
Ok, Fixed this with helper function which return `boolean`  and check that `boolean` in if